Formula/fio.rb

```diff
@@ -3,7 +3,7 @@ class Fio < Formula
   homepage "https://github.com/axboe/fio"
   url "https://github.com/axboe/fio/archive/fio-3.19.tar.gz"
   sha256 "809963b1d023dbc9ac7065557af8129aee17b6895e0e8c5ca671b0b14285f404"
-  license "GPL-2.0"
+  license "GPL-2.0-only"
```

Comment:
```suggestion
  license all_of: ["GPL-2.0-only", "GPL-2.0-or-later"]
```

Reply:
Sorry, I don't feel comfortable making this change. Fedora (a distro that cares about licensing) lists it as GPL v2 (https://src.fedoraproject.org/rpms/fio/blob/master/f/fio.spec#_6 ) so that's as far as I wish to go with this PR...

Reply:
It seems like Fedora doesn't have an `all_of` specifier either, so I'm not sure doing what Fedora does is necessarily appropriate here. Why does it make you uncomfortable? Is it inaccurate?

I don't feel comfortable stating that the software is `GPL-2.0-only` when in fact only parts of it are. Other Homebrew formulae use `all_of` and `any_of` to signify mixed licenses, so it would be misleading for this formula to falsely indicate that the license is not mixed.